bigtable/tests/system.py

```diff
@@ -36,6 +36,7 @@
 from test_utils.system import unique_resource_id
 
 
+PROJECT_ID = 'grass-clump-479'
```

Comment:
We derive the project ID from the environment / credentials -- it shouldn't be hardcoded.

Reply:
Fixed